zebra-scanner/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
```

Comment:
Adding another crate requires a new branch protection rule to make sure that crate builds individually.

Which admin will add that rule after the PR merges?

Reply:
I added a follow up task to the PR and crate for this.